fraud/interface.go

```diff
@@ -28,7 +30,7 @@ type Service interface {
 type Broadcaster interface {
 	// Broadcast takes a fraud `Proof` data structure that implements standard BinaryMarshal
 	// interface and broadcasts it to all subscribed peers.
-	Broadcast(context.Context, Proof) error
+	Broadcast(context.Context, Proof, ...pubsub.PubOpt) error
```

Comment:
Feel very conflicted about relying on PubSub structures in our interface. Is there a way to avoid it by publishing directly via pubsub accessed through service? We anyway convert the Service interface to a specific type, so that should be doable.

Reply:
yes, sure.